.github/workflows/main.yml

```diff
@@ -11,8 +11,8 @@ on:
       - '**/*.md'
 
 jobs:
-  ubuntu-latest-aurora-build-and-run-tests:
-    name: 'Build Driver And Run Tests'
+  ubuntu-latest-aurora-build:
```

Comment:
Building then not doing anything with it. Might as well remove this job

Reply:
I thought we want to reuse the artifacts built during test during release so we don't build it multiple times?

Reply:
Each job runs with a different runner. We're setting Java up and recloning the repo in each repo. Also, the test dockers compile again. I tried to get caching to work within the containers but wasn't able to get it to work.

Reply:
Okay I will just remove it then